test/core/rebalancingSetToken.spec.ts

```diff
@@ -478,6 +513,8 @@ contract('RebalancingSetToken', accounts => {
       const initialUnitShares = DEFAULT_UNIT_SHARES;
       const proposalPeriod = ONE_DAY_IN_SECONDS;
       const rebalanceInterval = ONE_DAY_IN_SECONDS;
+      const entranceFee = ZERO;
```

Comment:
Might be a bit of a nit but feel like these should be initialized with a non-zero value, because if nothing is assigned to the state var on the smart contract I believe it'll return 0. So I think to avoid an unlikely, though possible regression (the code looks fine) we should make this a positive number.